osf/models/registration_bulk_upload.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from django.db import models
+
+from osf.models.base import BaseModel
+from osf.utils.fields import NonNaiveDateTimeField
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationBulkUpload(BaseModel):
+    """Defines a database table of which each row stores information about one registration bulk upload.
+    """
+
+    # The hash of the CSV template payload
+    payload_hash = models.CharField(blank=False, null=False, unique=True, max_length=255)
+
+    # The status/state of the bulk upload
+    state = models.CharField(choices=[
+        ('pending', 'Database preparation in progress'),
```

Comment:
Minor nitpick: does it make more sense to have these as an enum?

Reply:
I don't have a preference. What are the pros and cons?

Reply:
If we have an enum defined somewhere, then we can import that enum elsewhere when we need to set the `state` field.

Reply:
Good point! Will do.